neovim/__init__.py

```diff
@@ -118,7 +118,7 @@ def setup_logging(name):
     if 'NVIM_PYTHON_LOG_FILE' in os.environ:
         prefix = os.environ['NVIM_PYTHON_LOG_FILE'].strip()
         major_version = sys.version_info[0]
-        logfile = '{0}_py{1}_{2}'.format(prefix, major_version, name)
+        logfile = '{}_py{}_{}'.format(prefix, major_version, name)
         handler = logging.FileHandler(logfile, 'w')
```

Comment:
I guess it can be useful in certain situations, e.g. something like:
```
x = ['a', 'b', 'c', 'd']
'{2}'.format(*x)

'real: {0.real}, imag: {0.imag}'.format(complex(2,3))
```

But yeah:

> Python has too many ways to do things.

But anyway, :+1: for this change, reads better without the numbers IMO.